data/discord_messages.yaml

```
@@ -22,43 +22,33 @@
     and also having enabled ZHA or deCONZ.
 format:
   description: "How to format text as code"
+  image: https://raw.githubusercontent.com/awesome-automations/hassbot-data/master/format.gif
   content: >-
     To format your text as code, enter three backticks on the first line, press
     Enter for a new line, paste your code, press Enter again for another new
-    line, and lastly three more backticks. [Here's an
-    example](https://raw.githubusercontent.com/awesome-automations/hassbot-data/master/format.gif)
+    line, and lastly three more backticks.
 
 
     Don't forget you can edit your post rather than repeatedly posting the same
     thing.
 
-
-    **For over 15 lines** you must use a code share site such as
-    <https://dpaste.org/> (pick _YAML_ for the language), <http://pastie.org/>
-    (pick _YAML_ for the language), or <https://paste.debian.net/> (pick _YAML_
-    for the language).
 rpi4:
   description: "RPi with SSD"
   content: >-
-    **Trying to install HassOS with an SSD in Rpi4?** Things you need to know:
-
+    **Trying to install HAOS with an SSD in Rpi4?** Things you need to know:
 
-    1) USB boot is supported only by the 64-bit [5.x dev
-    images](https://github.com/home-assistant/operating-system/releases) as of
-    now. (You installed a 5.x image but it's not working? -> Make sure it's
-    actually the 64-bit version!).
 
-    2)Make sure your Pi is updated to a stable
+    1)Make sure your Pi is updated to a stable
     [bootloader](https://www.raspberrypi.org/documentation/hardware/raspberrypi/bcm2711_bootloader_config.md#usbmassstorageboot)
     release that supports USB boot
 
-    3) If install still fails, then your SSD likely needs [quirks
+    2) If install still fails, then your SSD likely needs [quirks
     enabled](https://www.raspberrypi.org/forums/viewtopic.php?t=245931) to work
     correctly. Find your adapter's ID and add the quirks parameter to your
     `cmdline.txt` file (found in the boot partition) once you've flashed your
-    5.x HassOS image of choice on the SSD.
+    5.x HAOS image of choice on the SSD.
```

Should this be 5.x? We're at 9?
```suggestion
    HAOS image for your Raspberry Pi on the SSD.
```